README.md

```diff
@@ -257,8 +257,7 @@ Consider using [continuous integration](https://cloud.gov/docs/apps/continuous-d
 ### Recommendations
 
 1. You will probably want to connect your app to some kind of SMTP service to send transactional emails like password resets.
-1. The S3 Uploads plugin rewrites the URLs used by WordPress but does not flush the rewrite rules table automatically. To get around this, you can [run a task](https://cloud.gov/docs/getting-started/one-off-tasks/) to flush the rewrite rules after every `cf push` of your app. You can also automate those tasks by using [continuous integration](https://cloud.gov/docs/apps/continuous-deployment/).
-
+1. If you're running this application in production, you probably want to [connect a `redis32` service](https://cloud.gov/docs/services/redis/) to handle local session storage. Like most PHP applications, Wordpress stores session data (like permalinks and any other [transients](https://codex.wordpress.org/Transients_API) your app might use) to the local file system. As with uploads, these will get cleared periodically due to the ephemeral file system.
```

Comment:
Let's add a `cf create-service` command for the redis service instance and add it the example `manifest.yml`. Based on https://docs.cloudfoundry.org/buildpacks/php/gsg-php-sessions.html, it looks like the service name is significant unless we also set `REDIS_SESSION_STORE_SERVICE_NAME`, so it's important to either give guidance on the name or set the environment variable.